lib/ecto/multi.ex

```diff
@@ -602,6 +603,23 @@ defmodule Ecto.Multi do
   defp format_operation(other),
     do: other
 
+  @doc """
+  Adds a value to the changes so far under the given name.
+
+  ## Example
+
+      Ecto.Multi.new()
+      |> Ecto.Multi.put(:params, params)
+      |> Ecto.Multi.run()
```

Comment:
This looks like a stray bit from a previous revision that might want to be cleaned up in a small change.

Reply:
done on master, thank you for the PR!